e2e/cypress/integration/channel/remove_last_post_in_channel_spec.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Username of a test user that you want to start a DM with
+const username = 'aaron.ward';
```

Comment:
Instead of using an existing user with the API (see https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/support/api_commands.js#L493), create a new one to message so we don't have to worry about this test possibly impacting others or vice versa if this channel gets more messages sent to it.

Reply:
I implemented the changes that you have requested. Thanks for the feedback.